### PR TITLE
Add basic repo structure + doc

### PR DESCRIPTION
**Pull request by Allda**
The doc explains how to submit a new operator bundle and what's required
format of an operator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+done through Github pull requests which can be either open by the [CI pipeline][ci-pipeline] or
+manually by a user. The recommended way is to use the CI pipeline because it makes
+sure all necessary data are provided in the correct format. Documentation about how
+to use the pipeline is available in a repo.
+
+Branches in this repository (`stage`, `dev`, `qa`) correspond to preprod environments. If you want to test an operator submission in QA environment a pull request needs to be open against `qa` branch.
+
+## Directory structure
+This repository contains an `operator` directory where all certified operators
+are stored. Each operator has its directory with its name. Inside the directory,
```
Comment (amisstea) [inline README.md]: Let's clarify that it should match the "package name".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+# Redhat Marketplace operators pre-production catalog
+Pre-production catalog for Red Hat Marketplace Operator Bundles.
+
+# Operator bundle submission
+A new operator bundle submission needs to follow a predefined directory
+structure that is described below in this section. The new submission is
+done through Github pull requests which can be either open by the [CI pipeline][ci-pipeline] or
+manually by a user. The recommended way is to use the CI pipeline because it makes
+sure all necessary data are provided in the correct format. Documentation about how
+to use the pipeline is available in a repo.
```
Comment (amisstea) [inline README.md]: Small thing: `a repo` -> `the pipelines repo`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+    └── ci.yaml
+```
+
+### Format of ci.yaml file
+In the file, user needs to provide necessary details for the operator certification process.
+Currently, the certification process supports the following options:
+
+* `cert_project_id` - Identifier of certification project created through Red Hat Connect.
+* `merge` - A boolean value that determines whether a new operator is automatically
+merged when passed all required checks.
```
Comment (amisstea) [inline README.md]: Please make it clear that this is optional and what the default value is.

**Review by amisstea — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Allda) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (Allda) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (Allda) [reply README.md]: Fixed

**Review by amisstea — APPROVED** [review]
LGTM :+1:

**Pull request merged**